docs/user-guide/rules.md

```diff
@@ -0,0 +1,467 @@
+- [`selector-type-no-unknown`](../../lib/rules/selector-type-no-unknown/README.md): Disallow unknown type selectors.
+- [`unit-no-unknown`](../../lib/rules/unit-no-unknown/README.md): Disallow unknown units.
+
+## Enforce non-stylistic conventions
+
+You can enforce non-stylistic conventions with these `no-*` and `*-list` rules. They are powerful rules for making your CSS consistent. You'll need to configure most of them to suit your needs.
+
+### Allowed, disallowed & required
+
+Allow, disallow or require things with these `*-allowed-list`, `*-disallowed-list`, `*-required-list` and `no-*` rules.
+
+#### At-rule
+
+- [`at-rule-allowed-list`](../../lib/rules/at-rule-allowed-list/README.md): Specify a list of allowed at-rules.
+- [`at-rule-disallowed-list`](../../lib/rules/at-rule-disallowed-list/README.md): Specify a list of disallowed at-rules.
+- [`at-rule-no-vendor-prefix`](../../lib/rules/at-rule-no-vendor-prefix/README.md): Disallow vendor prefixes for at-rules (Autofixable).
+- [`at-rule-property-required-list`](../../lib/rules/at-rule-property-required-list/README.md): Specify a list of required properties for an at-rule.
+
+#### Color
+
+- [`color-hex-alpha`](../../lib/rules/color-hex-alpha/README.md): Require or disallow alpha channel for hex colors.
+- [`color-named`](../../lib/rules/color-named/README.md): Require (where possible) or disallow named colors.
+- [`color-no-hex`](../../lib/rules/color-no-hex/README.md): Disallow hex colors.
+
+#### Comment
+
+- [`comment-word-disallowed-list`](../../lib/rules/comment-word-disallowed-list/README.md): Specify a list of disallowed words within comments.
+
+#### Declaration
+
+- [`declaration-no-important`](../../lib/rules/declaration-no-important/README.md): Disallow `!important` within declarations.
+- [`declaration-property-unit-allowed-list`](../../lib/rules/declaration-property-unit-allowed-list/README.md): Specify a list of allowed property and unit pairs within declarations.
+- [`declaration-property-unit-disallowed-list`](../../lib/rules/declaration-property-unit-disallowed-list/README.md): Specify a list of disallowed property and unit pairs within declarations.
+- [`declaration-property-value-allowed-list`](../../lib/rules/declaration-property-value-allowed-list/README.md): Specify a list of allowed property and value pairs within declarations.
+- [`declaration-property-value-disallowed-list`](../../lib/rules/declaration-property-value-disallowed-list/README.md): Specify a list of disallowed property and value pairs within declarations.
+
+#### Function
+
+- [`function-allowed-list`](../../lib/rules/function-allowed-list/README.md): Specify a list of allowed functions.
+- [`function-disallowed-list`](../../lib/rules/function-disallowed-list/README.md): Specify a list of disallowed functions.
+- [`function-url-no-scheme-relative`](../../lib/rules/function-url-no-scheme-relative/README.md): Disallow scheme-relative urls.
```

Comment:
[note] Seeing the descriptions of `function-url-scheme-*`, using the word "URL" seems more consistent. Although this description change may be out of scope of this PR.

```suggestion
- [`function-url-no-scheme-relative`](../../lib/rules/function-url-no-scheme-relative/README.md): Disallow scheme-relative URLs.
```